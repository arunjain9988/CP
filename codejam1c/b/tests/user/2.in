5
111231321364654163
2121310
284658698
9999999
42125131