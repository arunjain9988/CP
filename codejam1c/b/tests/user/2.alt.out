Case #1: 1234567891011121314
Case #2: 2345678
Case #3: 285286287
Case #4: 10001001
Case #5: 42134214
